### ISSUE TITLE
False positive of  `ObjectCalisthenics\Sniffs\CodeAnalysis\OneObjectOperatorPerLineSniff`

### ISSUE THREAD

**Issue by TomasVotruba** [object-calisthenics/phpcs-calisthenics-rules#82]
This one shall not be reported, as it is part of fluent interface, that is described in map of methods.

```php
/**
 * @return TrainingTerm[]
 */
public function fetchPast(): array
{
    return $this->entityRepository->createQueryBuilder('tt')
        ->where('tt.endDateTime < CURRENT_DATE()')
        ->getQuery()
        ->getResult();
}
```

**Comment by amandiobm** [comment]
In my case I have

```
/**
 * Creates the application.
 *
 * @return \Illuminate\Foundation\Application
 */
public function createApplication()
{
    $app = require __DIR__.'/../bootstrap/app.php';

    $app->make(Kernel::class)->bootstrap();

    return $app;
}
```

that should be correct, but I changed to 

```
$app->make(Kernel::class)
    ->bootstrap();
```

Still giving me an error.

Any ideas?

**Comment by TomasVotruba** [comment]
Outdated, needs failing test case + fix
